src/Data/List/Relation/Unary/All/Properties.agda

```diff
@@ -388,6 +389,18 @@ mapMaybe⁺ {xs = x ∷ xs} {f = f} (px ∷ pxs) with f x
 ... | just v with px
 ...   | just pv = pv ∷ mapMaybe⁺ pxs
 
+------------------------------------------------------------------------
+-- catMaybes
+
+catMaybesAll⁺ : All (Maybe.All P) xs → All P (catMaybes xs)
+catMaybesAll⁺ [] = []
+catMaybesAll⁺ (just px ∷ pxs) = px ∷ catMaybesAll⁺ pxs
+catMaybesAll⁺ (nothing ∷ pxs) = catMaybesAll⁺ pxs
+
+catMaybesAny⁺ : All (Maybe.Any P) xs → All P (catMaybes xs)
```

Comment:
I think the names for these lemmas should be `Any-catMaybes⁺` and `All-catMaybes⁺`

Reply:
I renamed it.